app/views/travel_expense_reports/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for(@travel_expense_report) do |f| %>
+<%= form_for @travel_expense_report,:html => { :class => "form-horizontal travel_expense_report" } do |f| %>
```

Comment:
Könntest du das hier in die Punktnotation umändern? (also html: {...} anstatt :html => {...})